python/ray/serve/_private/autoscaling_state.py

```diff
@@ -227,6 +237,13 @@ def record_request_metrics_for_replica(
         ):
             self._replica_requests[replica_id] = ReplicaMetricReport(
                 running_requests=window_avg,
+                metrics=None,
+                timestamp=send_timestamp,
+            )
+
+            self._replica_metrics[replica_id] = ReplicaMetricReport(
+                running_requests=None,
+                metrics=metrics,
                 timestamp=send_timestamp,
             )
```

Comment:
for custom autoscaling, will request metrics ever be used together with custom metrics? why keep both _replica_requests and _replica_metrics instead of merging?